dev/com.ibm.websphere.appserver.features/visibility/public/ejbRemote-3.2/com.ibm.websphere.appserver.ejbRemote-3.2.feature

```diff
@@ -4,7 +4,61 @@ WLP-DisableAllFeatures-OnConflict: false
 visibility=public
 IBM-App-ForceRestart: install, \
  uninstall
-IBM-API-Package: com.ibm.ws.ejb.portable; type="internal"
+IBM-API-Package: com.ibm.ws.ejb.portable; type="internal", \
+  com.ibm.websphere.endpoint; type="ibm-api", \
+  javax.rmi; type="spec", \
```

Comment:
I chose to do that because they are different.  The ejbLite / enterpriseBeansLite has `require-java:="9"` and the ejbRemote / enterpriseBeansRemote does not.

Reply:
Ah yes, I missed that subtle difference.